cookbook/frontend/bower.rst

```diff
@@ -0,0 +1,125 @@
+.. index::
+    single: Front-end; Bower
+
+Using Bower with Symfony
+========================
+
+Symfony and all its packages are perfectly managed by Composer. Bower is a
+dependency management tool for front-end dependencies, like Twitter Bootstrap
+or jQuery. Since Symfony isn't purely a back-end framework, it can't deliver
+you much help with Bower. Fortunately, it is very easy to use!
+
+Installing Bower
+----------------
+
+Bower_ is build on top of NodeJS_. Make sure you have that installed and
```

Comment:
You're correct (at least, their website uses the Node.js notation). I'll update it